modules/apps/object/object-web/src/test/java/com/liferay/object/web/internal/display/layout/ObjectEntryDisplayContextImplTest.java

```diff
@@ -0,0 +1,123 @@
+ */
+
+package com.liferay.object.web.internal.display.layout;
+
+import com.liferay.dynamic.data.mapping.expression.DDMExpressionFactory;
+import com.liferay.dynamic.data.mapping.form.renderer.DDMFormRenderer;
+import com.liferay.item.selector.ItemSelector;
+import com.liferay.object.constants.ObjectWebKeys;
+import com.liferay.object.field.business.type.ObjectFieldBusinessType;
+import com.liferay.object.field.business.type.ObjectFieldBusinessTypeRegistry;
+import com.liferay.object.internal.field.business.type.DateTimeObjectFieldBusinessType;
+import com.liferay.object.model.ObjectField;
+import com.liferay.object.rest.manager.v1_0.ObjectEntryManagerRegistry;
+import com.liferay.object.scope.ObjectScopeProviderRegistry;
+import com.liferay.object.service.ObjectDefinitionLocalService;
+import com.liferay.object.service.ObjectEntryLocalService;
+import com.liferay.object.service.ObjectEntryService;
+import com.liferay.object.service.ObjectFieldLocalService;
+import com.liferay.object.service.ObjectLayoutLocalService;
+import com.liferay.object.service.ObjectRelationshipLocalService;
+import com.liferay.object.web.internal.object.entries.display.context.ObjectEntryDisplayContextImpl;
+import com.liferay.portal.kernel.test.ReflectionTestUtil;
+import com.liferay.portal.test.rule.LiferayUnitTestRule;
+
+import jakarta.servlet.http.HttpServletRequest;
+
+import org.junit.Assert;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+import org.mockito.Mockito;
+
+/**
+ * @author Aquiles Duarte
+ */
+public class ObjectEntryDisplayContextImplTest {
+
+	@ClassRule
+	public static final LiferayUnitTestRule liferayUnitTestRule =
+		LiferayUnitTestRule.INSTANCE;
+
+	@Test
+	public void testSystemFieldReturnsDateTimeBusinessType() {
+		Mockito.when(
+			_httpServletRequest.getAttribute(
+				ObjectWebKeys.OBJECT_ENTRY_READ_ONLY)
+		).thenReturn(
+			Boolean.FALSE
+		);
+
+		ObjectField objectField = Mockito.mock(ObjectField.class);
+
+		Mockito.when(
+			objectField.getBusinessType()
+		).thenReturn(
+			"Date"
+		);
+
+		Mockito.when(
+			objectField.getName()
+		).thenReturn(
+			"createDate"
+		);
+
+		DateTimeObjectFieldBusinessType dateTimeBusinessType =
+			new DateTimeObjectFieldBusinessType();
+
+		Mockito.when(
+			_objectFieldBusinessTypeRegistry.getObjectFieldBusinessType(
+				"DateTime")
+		).thenReturn(
+			dateTimeBusinessType
+		);
+
+		ObjectEntryDisplayContextImpl objectEntryDisplayContextImpl =
+			new ObjectEntryDisplayContextImpl(
+				_ddmExpressionFactory, _ddmFormRenderer, _httpServletRequest,
+				_itemSelector, _objectDefinitionLocalService,
+				_objectEntryManagerRegistry, _objectEntryLocalService,
+				_objectEntryService, _objectFieldBusinessTypeRegistry,
+				_objectFieldLocalService, _objectLayoutLocalService,
+				_objectRelationshipLocalService, _objectScopeProviderRegistry);
+
+		ObjectFieldBusinessType result = ReflectionTestUtil.invoke(
```

Comment:
This is ok, but maybe it would be better to have a integration test in `ObjectEntryDisplayContextTest` class to test the method `ObjectEntryDisplayContextImpl#renderDDMForm`. What do you think, is it feasible? Can you try?